src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_functionapp_commands.py

```diff
@@ -1082,8 +1082,9 @@ def test_functionapp_with_default_distributed_tracing(self, resource_group, stor
         self.cmd('functionapp create -g {} -n {} -p {} -s {} --runtime java --functions-version 4'
                  .format(resource_group, functionapp_name, plan, storage_account))
 
-        app_set = self.cmd('functionapp config appsettings list -g {} -n {}'.format(resource_group,
-                                                                                    functionapp_name)).get_output_in_json()
+        self.cmd('functionapp config appsettings set -g {} -n {} --settings FOO=BAR'.format(resource_group, functionapp_name))
```

Comment:
For this `test_functionapp_with_default_distributed_tracing` unit test, if I run it `--live` then I do not get any errors but for some reason, if I do not have this line, then it does not find the appsetting in L1088 even though it is listed in the recording. Do you know why this succeeds when running the test `--live` but then fails when rerunning the recording without `--live`?

Reply:
In addition, we will launch the release for Ignite Sprint the day after tomorrow. If this issue is only related to testing and does not block merge PR, then we can merge this PR first and investigate the testing issue in the future

Reply:
@zhoxing-ms @yanzhudd Thank you. Yes, I do not think this should block merging the PR. The issue I have is only related to testing when using the recording (even though the recording has the app settings I am validating for), but it works when I test it manually or when I run the tests in `--live` mode. As such, can we please merge the PR? Thank you.